config/nginx-config/nginx-wp-common.conf

```diff
@@ -79,3 +79,5 @@ location ~ \.php$ {
     # And get to serving the file!
     fastcgi_index  index.php;
 }
+
+include /etc/nginx/custom-rules/*.conf;
```

Comment:
My understanding is that this is included by all site nginx configs. I don't think that this is needed as a site template can already do this via `vvv-nginx.conf`

Reply:
I implemented this one for https://github.com/Varying-Vagrant-Vagrants/custom-site-template/pull/5
But with that we cannot load on specific site, but as you said for this case is not the best solution.
I was thinking if someone want a service that can inject their own rules on a website, with this can be possible. For this idea I don't have an example now but probably is better to remove it.